pandas/tslib.pyx

```diff
@@ -885,7 +885,12 @@ def _make_nan_func(func_name):
     f.__name__ = func_name
     return f
 
+
+# GH14940
+_round_methods = ['round', 'floor', 'ceil']
+
 _nat_methods = ['date', 'now', 'replace', 'to_pydatetime', 'today']
+_nat_methods.extend(_round_methods)
```

Comment:
you can just directly add these I think, no need to have a separate _round_methods